example/demo/docker-compose.yaml

```diff
@@ -0,0 +1,90 @@
+version: '1'
+services:
+  dbzui-zookeeper:
+    container_name: dbzui-zookeeper
+    image: debezium/zookeeper:${DEBEZIUM_VERSION}
+    networks:
+      - dbzui-network
+  dbzui-kafka:
+    container_name: dbzui-kafka
+    image: debezium/kafka:${DEBEZIUM_VERSION}
+    ports:
+      - "9092:9092"
+    depends_on:
+      - dbzui-zookeeper
+    environment:
+      - ZOOKEEPER_CONNECT=dbzui-zookeeper:2181
+    networks:
+      - dbzui-network
+  dbzui-db-pg:
+    container_name: dbzui-db-pg
+    image: debezium/example-postgres:${DEBEZIUM_VERSION}
+    ports:
+      - "65432:5432"
+    environment:
+      - POSTGRES_USER=postgres
+      - POSTGRES_PASSWORD=postgres
+    networks:
+      - dbzui-network
+  dbzui-db-mysql:
+    container_name: dbzui-db-mysql
+    image: debezium/example-mysql:${DEBEZIUM_VERSION}
+    ports:
+      - "63306:3306"
+    environment:
+      - MYSQL_ROOT_PASSWORD=debezium
+      - MYSQL_USER=mysqluser
+      - MYSQL_PASSWORD=mysql
+    networks:
+      - dbzui-network
+  dbzui-db-mongo:
+    container_name: dbzui-db-mongo
+    image: mongo:3.6
+    command: ["mongod", "--replSet", "rs0", "--auth"]
+    ports:
+      - "37017:27017"
+    volumes:
+    - ./backend/src/test/resources/initialize-mongo-single.js:/docker-entrypoint-initdb.d/initialize-mongo-single.js
+    environment:
+      - MONGO_INITDB_ROOT_USERNAME=admin
+      - MONGO_INITDB_ROOT_PASSWORD=admin
+      - MONGO_INITDB_DATABASE=admin
+    networks:
+      - dbzui-network
+  mongo-initializer:
+    image: mongo:3.6
+    networks:
+      - dbzui-network
+    depends_on:
+      - dbzui-db-mongo
+    command: bash -c 'sleep 20 && mongo -u admin -p admin --authenticationDatabase admin dbzui-db-mongo:27017/inventory --eval "rs.initiate();"'
+  dbzui-connect:
+    container_name: dbzui-connect
+    image: debezium/connect:${DEBEZIUM_VERSION}
+    ports:
+      - "8083:8083"
+    depends_on:
+      - dbzui-kafka
+      - dbzui-db-pg
+    environment:
+      - BOOTSTRAP_SERVERS=dbzui-kafka:9092
+      - GROUP_ID=1
+      - CONFIG_STORAGE_TOPIC=my_connect_configs
+      - OFFSET_STORAGE_TOPIC=my_connect_offsets
+      - STATUS_STORAGE_TOPIC=my_connect_statuses
+    networks:
+      - dbzui-network
+  dbzui-debezium-ui:
```

Comment:
Do we need the "dbzui-" prefix in all the service names? Seems it could simply be omitted?

Reply:
yeah, I think it can be omitted